go/sample/main.go

```diff
@@ -33,7 +33,7 @@ func main() {
 	hk := flag.String("hyperkit", "", "HyperKit binary to use")
 	statedir := flag.String("state", "", "Directory to keep state in")
 	vpnkitsock := flag.String("vpnkitsock", "auto", "Path to VPNKit socket")
-	disk := flag.String("disk", "", "Path to disk image")
+	disks := flag.String("disk", "", "Comma separated list of disk images, format [image]{:size_in_MB}. If size is specified the image will be created if it doesn't exist. If image name is omitted a default name will be used.")
```

Comment:
Could we maybe make this the same as/similar to the LinuxKit command line, as in https://github.com/linuxkit/linuxkit/blob/master/src/cmd/linuxkit/run_qemu.go#L64

this also has a more elgant way for parsing the command line in: https://github.com/linuxkit/linuxkit/blob/master/src/cmd/linuxkit/util.go#L153 so the code below can be moved into a method